### PR TITLE
chore: retire mtls-abassador welcome mender-gateway

### PR DESCRIPTION
**Pull request by merlin-northern**
goes with https://github.com/mendersoftware/deviceauth-enterprise/pull/279
pipeline: https://gitlab.com/Northern.tech/Mender/mender-qa/-/pipelines/1278559650

Ticket: MEN-5307

### CODE REVIEW

File: component-maps.yml [hunk 1]
```diff
@@ -218,13 +218,6 @@ git:
     - mender-auditlogs
     release_component: true
 
-  mtls-ambassador:
-    docker_image:
-    - mtls-ambassador
-    docker_container:
-    - mtls-ambassador
-    release_component: true
```
Comment (lluiscampos) [inline component-maps.yml:226]: Please keep the component and just set `release_component` to `false` :+1:

**Comment by lluiscampos** [comment]
@merlin-northern I am working on this. Enjoy your time off and we sync back next Monday.

File: component-maps.yml (hunk 1 above)
Comment (merlin-northern) [reply component-maps.yml:226]: thanks amigo for early and unasked for review :* you always help

Comment (merlin-northern) [reply component-maps.yml:226]: fixed.

**Comment by lluiscampos** [comment]
Closing in favor of #2603

**Pull request closed without merging**